thirdeye-ui/docker/templates/default.conf.template

```diff
@@ -29,5 +29,7 @@ server {
     }
     location / {
         try_files $uri $uri/ /index.html;
+        expires 1h;
+        add_header Cache-Control "no-cache ";
```

Comment:
Will this disable cache for everything. I also found this:
https://stackoverflow.com/questions/41631399/disable-caching-of-a-single-file-with-try-files-directive

Reply:
Made changes based on the stackoverflow